resources/views/layouts/app.blade.php

```diff
@@ -30,7 +30,7 @@
   <link rel="manifest" href="{{ asset('/site.webmanifest') }}">
   <link rel="mask-icon" href="{{ asset('/safari-pinned-tab.svg') }}" color="#3d8fcc">
   <script defer src="{{ asset(mix('js/manifest.js')) }}"></script>
-  <script defer src="{{ asset(mix('js/vendor.js')) }}"></script>
+  <!-- <script defer src="{{ asset(mix('js/vendor.js')) }}"></script> -->
```

Comment:
I noticed this was also commented out. Is the vendor JavaScript no longer split into another file?

Reply:
I can see the use case but it was not being compiled, so laravel threw an error when it couldn't be found.

Reply:
It looks like the extract function filtered down to `@barba/core` doesn't produce a vendor file, but a general `.extract()` method call does. https://github.com/Calinou/godot-asset-library-laravel/blob/master/webpack.mix.js#L20

Not sure why that's happening. I guess this is a separate issue at this point.